test/integration/util/util.go

```diff
@@ -41,6 +42,10 @@ type OcRunner struct {
 	CommandPath string
 }
 
+const (
+	OpenShiftVersion = "v1.4.1" // Should match OPENSHIFT_VERSION in Makefile
```

Comment:
Then, we can have regex pattern checking to verify the content of json file for oc path as

`MACHINE_HOME_DIR/cache/oc/v(some regex pattern)/oc`